rln-js/index.html

```diff
@@ -432,7 +432,7 @@ <h4 class="mu1">Messages</h4>
     sendButton.onclick = async () => {
         const text = utils.utf8ToBytes(textInput.value);
         const timestamp = new Date();
-        const msg = ProtoChatMessage.create({text, nick: nicknameInput.value, timestamp: timestamp.valueOf()});
+        const msg = ProtoChatMessage.create({text, nick: nicknameInput.value, timestamp: Math.floor(timestamp.valueOf() / 1000)});
```

Comment:
This is the timestamp for the chat message object, it's not the waku message one.

Reply:
What are you trying to fix?